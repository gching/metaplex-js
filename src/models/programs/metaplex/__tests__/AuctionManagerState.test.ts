import {
  AuctionManagerState,
  AuctionManagerStatus,
} from '../AuctionManagerState';

describe('AuctionManagerState', () => {
  it('should instantiate with the correct values', () => {
    const auctionManagerState = new AuctionManagerState({
      winningConfigItemsValidated: 1,
      status: AuctionManagerStatus.Finished,
      winningConfigStates: [],
    });

    expect(auctionManagerState).toMatchSnapshot();
  });
});
