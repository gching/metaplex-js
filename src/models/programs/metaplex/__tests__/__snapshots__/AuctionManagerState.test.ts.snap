// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuctionManagerState should instantiate with the correct values 1`] = `
AuctionManagerState {
  "participationState": undefined,
  "status": 4,
  "winningConfigItemsValidated": 1,
  "winningConfigStates": Array [],
}
`;
